src/routes/JobOpportunity/Overview.tsx

```diff
@@ -86,137 +111,103 @@ const Overview = () => {
         </SimpleNavigationList>
       }
     >
+      {loginModalOpen && <LoginModal onClose={() => setLoginModalOpen(false)} isOpen={loginModalOpen} />}
       <Title value={title ?? ''} />
       <h1 className="mb-5 text-heading-2 sm:text-heading-1">{title}</h1>
-      <Tabs />
-      <div className="flex flex-col gap-11">
+      <div className="flex flex-row gap-7 my-8">
```

Comment:
Hide action buttons in print view
```suggestion
      <div className="flex flex-row gap-7 my-8 print:hidden">
```